docs/se/guides/health.adoc

```diff
@@ -303,6 +352,40 @@ ObserveFeature observe = ObserveFeature.builder()
+
+==== Adding configuration to a custom observer
+In addition to preparing the health observer builder with hard-coded settings, your code can also apply configuration for health so your user could set any health behavior there as well.
+
+The generated `Main` class in the application already creates a `Config` object for the top-level config node. Using the following code to create the observe feature also applies any health-related configuration settings to the custom health observer. Notice the added line just before the `HealthObserver.Build` `build()` invocation near the end of the example code.
+
+[source.java]
+.Apply health configuration to your custom health observer
+----
+ObserveFeature observe = ObserveFeature.builder()
+        .config(config.get("server.features.observe"))
+        .addObserver(HealthObserver.builder()
+                             .endpoint("/myhealth")
+                             .details(true)
+                             .addCheck(() -> HealthCheckResponse.builder()
+                                               .status(System.currentTimeMillis() - serverStartTime.get() >= 8000)
+                                               .detail("time", System.currentTimeMillis())
+                                               .build(),
+                                       HealthCheckType.STARTUP,
+                                       "warmedUp")
+                             .config(config.get("server.features.observe.observers.health")) // <1>
+                             .build()
+        )
+        .build();
+----
+<1> Find and apply any health-related settings from configuration at the `server.features.observe.observers.health` config key.
+
+Your code decides what config key to use for retrieving the configuration. Recall earlier, before adding custom health checks, you added a config section for health--to set `details` to `true`--at `server.features.observe.observers.health`. Helidon used that configuration to set up the health observer _it_ created automatically. To be consistent for your users, it's a good idea for your application code, as it preares a custom `HealthObserver`, to look in the same place Helidon does for health config.
+
+Order is important. Here, the code first sets `details` to `true` explicitly and later applies configuration. If your end user sets `details` in the `server.features.observe.observers.health` config to `false`, that setting overrides the hard-coded `true` setting in the code _because of where in the code you apply the configuration_. Try changing the `details` value to `false` in the config file and then stop, rebuild, and rerun the application. Access the health endpoint and notice that the output is no longer detailed.
+
+In general, most applications should apply settings from config _after_ assigning any settings in the code so users have the final say, but there might be exceptions in your particular case.
```

Comment:
I see the "users" again. I'd would use "external config" or simply "config" instead, but I understand the notion better now.

Reply:
Rephrased this a bit.